web/src/routes/SearchPage.tsx

```diff
@@ -99,20 +118,25 @@ const SearchPage = ({ city, cityCode, languageCode, pathname }: CityRouteProps):
         spaceSearch
       />
       <List>
-        {searchResults.map(({ category, contentWithoutHtml }) => (
-          <SearchListItem
-            key={category.path}
-            query={query}
-            category={category}
-            contentWithoutHtml={contentWithoutHtml}
-          />
-        ))}
+        {(query.length !== 0 ? results : allPossibleResults).map(
+          ({ title, id, content, location, path, url, thumbnail }) => (
+            <SearchListItem
```

Comment:
The sames goes for this one. If we add the component to the minisearch search result this should be slightly faster.